Missing link to windowsapp.lib?

When I tried to install version 5.1.0 in vcpkg, I encountered the following error:

```
CMakeFiles\libremidi.dir/manifest.res" failed (exit code 1120) with the following output:
   Creating library libremidi.lib and object libremidi.exp
midi_in.cpp.obj : error LNK2019: unresolved external symbol WINRT_IMPL_RoGetActivationFactory referenced in function "struct winrt::hresult __cdecl winrt::impl::get_runtime_activation_factory_impl<0>(struct winrt::param::hstring const &,struct winrt::guid const &,void * *)" (??$get_runtime_activation_factory_impl@$0A@@impl@winrt@@YA?AUhresult@1@AEBUhstring@param@1@AEBUguid@1@PEAPEAX@Z)
midi_out.cpp.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoGetActivationFactory
observer.cpp.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoGetActivationFactory
midi_in.cpp.obj : error LNK2019: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException referenced in function "private: void __cdecl winrt::hresult_error::originate(struct winrt::hresult,void *,struct std::source_location const &)" (?originate@hresult_error@winrt@@AEAAXUhresult@2@PEAXAEBUsource_location@std@@@Z)

midi_out.cpp.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException

observer.cpp.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException

libremidi.dll : fatal error LNK1120: 2 unresolved externals

ninja: build stopped: subcommand failed.
```

--- a/cmake/libremidi.winmidi.cmake
+++ b/cmake/libremidi.winmidi.cmake
@@ -66,3 +66,6 @@ target_include_directories(libremidi SYSTEM ${_public}
 target_compile_definitions(libremidi ${_public} LIBREMIDI_WINMIDI)
 set(LIBREMIDI_HAS_WINMIDI 1)
 target_link_libraries(libremidi INTERFACE RuntimeObject)
+if(MSVC)
+  target_link_libraries(libremidi ${_public} windowsapp)
+endif()

--- a/cmake/libremidi.winuwp.cmake
+++ b/cmake/libremidi.winuwp.cmake
@@ -23,6 +23,7 @@ if(CPPWINRT_PATH)
   # We don't need /ZW option here (support for C++/CX)' as we use C++/WinRT
   if(MSVC)
     target_compile_options(libremidi ${_public} /EHsc)
+    target_link_libraries(libremidi ${_public} windowsapp)
   endif()
 else()
   message(STATUS "libremidi: Failed to find Windows SDK, UWP MIDI backend will not be available")

